sql.txt

```diff
@@ -1,14 +1,23 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 10 OFFSET 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
+SELECT * FROM Customers WHERE ContactName LIKE ("A%");
 
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND Quantity > 10;
+
 
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
-corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below. 
+INSERT INTO Products VALUES (NULL, "pumpkin juice", 1,1, "12 - 550 ml bottles", 5);
+INSERT INTO Products VALUES (NULL, "butter beer", 1,1, "15- 500 ml bottles", 7);
+INSERT INTO Products VALUES (NULL, "dragon vodka", 1,1, "42 - 750 ml bottles", 20);
```

Comment:
These commands work individually. But I would suggest using the simpler syntax for example:
INSERT INTO table_name (column_list) VALUES (value_list_1), (value_list_2), (value_list_3);
But great job keep it up.